samples/sampler/src/ThemeEnvironment/ThemeEnvironment.module.less

```diff
@@ -6,9 +6,6 @@
 // from ~@enact/ui/styles/core.less. Can't import global mixins.
 .enact-selectable() {
 	cursor: auto;
-	-ms-user-select: element;
-	-webkit-user-select: text;
```

Comment:
As I mentioned in the enact PR, I think we should keep `-webkit-` prefix one.

Reply:
OK.